### ISSUE TITLE
Octave installed but command is broken

### ISSUE THREAD

**Issue by nazavode** [Homebrew/homebrew-science#5116]
Installation of Octave formula went smoothly, but every time I try to run the octave command from terminal, I end up with this:

```
# octave
dyld: Library not loaded: /usr/local/opt/suite-sparse/lib/libsuitesparseconfig.4.5.3.dylib
  Referenced from: /usr/local/bin/octave
  Reason: image not found
[1]    4664 trace trap  octave
```

#### The complete command that you issued:

```
brew cask install xquartz
brew tap homebrew/science
brew update && brew upgrade
brew install octave
```

#### Gist with output of `brew doctor`:

```
# brew doctor
Your system is ready to brew.
```

#### Link output by `brew gist-logs <formula>`:

```
# brew gist-logs octave
https://gist.github.com/3ed8343b7e1ae468b7f1795fc4f5da1e
```

#### Output of `brew config`:

```
# brew config
HOMEBREW_VERSION: 1.1.9-138-g674e5f13f
ORIGIN: https://github.com/Homebrew/brew.git
HEAD: 674e5f13f8e30dedee92abffa882406c883948d9
Last commit: 47 minutes ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: bdfa7525db2dd8f5ef3e3a3349d3b1301c298362
Core tap last commit: 2 hours ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_REPOSITORY: /usr/local/Homebrew
HOMEBREW_CELLAR: /usr/local/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://homebrew.bintray.com
CPU: quad-core 64-bit haswell
Homebrew Ruby: 2.0.0-p648
Clang: 8.0 build 800
Git: 2.11.1 => /usr/local/bin/git
Perl: /usr/bin/perl
Python: /usr/local/bin/python => /usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/bin/python2.7
Ruby: /usr/bin/ruby => /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby
Java: 1.6.0_65-b14-468
macOS: 10.11.6-x86_64
Xcode: N/A
CLT: 8.2.0.0.1.1480973914
X11: 2.7.11 => /opt/X11
```

**Comment by ilovezfs** [comment]
Yeah you need to rebuild from source or wait a bit.

See https://github.com/Homebrew/homebrew-science/pull/5112 which I'm still working on.

**Comment by nazavode** [comment]
Yep, thank you very much, looked around quite a bit for some help but didn't manage to find that issue, sorry.

**Comment by ilovezfs** [comment]
Closed via #5112 which I've now shipped.
